packages/sui-deploy/src/utils.js

```diff
@@ -19,7 +19,10 @@ const getDeployNameFromProgram = async program => {
   const deployName =
     deployBaseName +
     (program.branch
-      ? '-' + (process.env.TRAVIS_BRANCH || (await getGitBranch()))
+      ? '-' +
```

Comment:
Why instead of couple our tool with 3rd party integrations we don't decide to pass the 'deploy alias' by param?

Something like `sui-deploy ... ... --deployName=$TRAVIS_PULL_REQUEST`

Doing that we are avoiding be coupled with TRAVIS on an open source tool.

What do you think?

Reply:
@ddmarin94 Talking with carlos we decided to add a CLI option in order to allow our users to decide which name of branch are they going to use as I added. In order to maintain the retrocompatibility with our current users we will don't remove the travis env var.

Reply:
The first arg is already the deployName...
Guys, this "option" already exists:
```sh
sui-deploy "milanuncios-$TRAVIS_PULL_REQUEST" --now
```

Reply:
Plus, if you have to to sonething, move `process.env.TRAVIS_BRANCH` logic into getGitBranch func...

Reply:
Wow you are right! So sorry about that. I am agree to remove the travis branch in order to pass it as an option

Reply:
This pr will be closed, we found a different way to solve this problem.